### PR TITLE
Update memory dump command with download options

### PR DESCRIPTION
**Pull request by ShellyKa13**
Signed-off-by: Shelly Kagan <skagan@redhat.com>

### CODE REVIEW

File: docs/operations/memory_dump.md [hunk 1]
```diff
@@ -57,6 +57,21 @@ $ virtctl memory-dump get my-vm --claim-name=new-pvc --create-claim
 
 A PVC with size big enough for the dump will be created. We can also request specific storage class and access mode with appropriate flags.
 
+#### Download memory dump
+By adding `--output` flag the memory will be dumped to the pvc and then downloaded to the given output path.
```
Comment (cwilkers) [inline docs/operations/memory_dump.md:61]:
```suggestion
By adding the `--output` flag, the memory will be dumped to the PVC and then downloaded to the given output path.
```

File: docs/operations/memory_dump.md [hunk 2]
```diff
@@ -57,6 +57,21 @@ $ virtctl memory-dump get my-vm --claim-name=new-pvc --create-claim
...
+
+```bash
+$ virtctl memory-dump get myvm --claim-name=memoryvolume --create-claim --output=memoryDump.dump.gz
+```
+
+For downloading the last memory dump from the PVC associated to the VM, without triggering another memory dump use the memory dump donwload command
```
Comment (cwilkers) [inline docs/operations/memory_dump.md:67]:
```suggestion
For downloading the last memory dump from the PVC associated with the VM, without triggering another memory dump, use the memory dump download command.
```

File: docs/operations/memory_dump.md [hunk 3]
```diff
@@ -57,6 +57,21 @@ $ virtctl memory-dump get my-vm --claim-name=new-pvc --create-claim
...
+
+```bash
+$ virtctl memory-dump download myvm --output=memoryDump.dump.gz
+```
+
+For downloading memory dump from a PVC already removed from the VM association you can use the [virtctl vmexport command](https://github.com/kubevirt/user-guide/blob/main/docs/operations/export_api.md)
```
Comment (cwilkers) [inline docs/operations/memory_dump.md:73]:
```suggestion
For downloading a memory dump from a PVC already removed from the VM association you can use the [virtctl vmexport command](https://github.com/kubevirt/user-guide/blob/main/docs/operations/export_api.md)
```

Also, would it make sense to rephrase "removed from the VM association" as "disassociated from the VM" ?

**Review by cwilkers — CHANGES_REQUESTED** [review]
Great feature, thanks for your documentation PR!

I have only minor grammatical edits

**Comment by ShellyKa13** [comment]
@cwilkers thanks for the review, fixed everything according to your suggestions.

**Pull request merged**